integration/v7/isolated/services_command_test.go

```diff
@@ -212,3 +212,137 @@ var _ = Describe("services command", func() {
+				Expect(session.Out).To(matchHelpMessage)
+			})
+		})
+	})
+
+	Context("has no services", func() {
+		BeforeEach(func() {
+			helpers.LoginCF()
+			helpers.TargetOrgAndSpace(ReadOnlyOrg, ReadOnlySpace)
+		})
+
+		It("tells the user 'no services found'", func() {
+			session := helpers.CF(command)
+			Eventually(session).Should(Exit(0))
+
+			Expect(session).To(SatisfyAll(
+				Say(`Getting service instances in org %s / space %s as %s...\n`, ReadOnlyOrg, ReadOnlySpace, userName),
+				Say(`No service instances found\.\n`),
+			))
+		})
+	})
+
+	Context("has services and applications", func() {
+		var (
+			orgName   string
+			spaceName string
+
+			broker *servicebrokerstub.ServiceBrokerStub
+
+			managedService1      string
+			managedService2      string
+			userProvidedService1 string
+			userProvidedService2 string
+			//appName1             string
+			//appName2             string
+		)
+
+		BeforeEach(func() {
+			orgName = helpers.NewOrgName()
+			spaceName = helpers.NewSpaceName()
+			helpers.SetupCF(orgName, spaceName)
+
+			userProvidedService1 = helpers.PrefixedRandomName("UPS1")
+			userProvidedService2 = helpers.PrefixedRandomName("UPS2")
+			Eventually(helpers.CF("cups", userProvidedService1, "-p", `{"username": "admin", "password": "admin"}`)).Should(Exit(0))
+			Eventually(helpers.CF("cups", userProvidedService2, "-p", `{"username": "admin", "password": "admin"}`)).Should(Exit(0))
+
+			broker = servicebrokerstub.New().WithPlans(2).EnableServiceAccess()
+			managedService1 = helpers.PrefixedRandomName("MANAGED1")
+			managedService2 = helpers.PrefixedRandomName("MANAGED2")
+			helpers.CreateManagedServiceInstance(broker.FirstServiceOfferingName(), broker.FirstServicePlanName(), managedService1)
+
+			broker.Services[0].Plans[0].MaintenanceInfo = &config.MaintenanceInfo{Version: "2.0.0"}
+			broker.Configure().Register()
+			helpers.CreateManagedServiceInstance(broker.FirstServiceOfferingName(), broker.FirstServicePlanName(), managedService2)
+
+			//appName1 = helpers.PrefixedRandomName("APP1")
+			//appName2 = helpers.PrefixedRandomName("APP2")
+			//helpers.WithHelloWorldApp(func(appDir string) {
+			//	Eventually(helpers.CF("push", appName1, "--no-start", "-p", appDir, "-b", "staticfile_buildpack", "--no-route")).Should(Exit(0))
+			//	Eventually(helpers.CF("push", appName2, "--no-start", "-p", appDir, "-b", "staticfile_buildpack", "--no-route")).Should(Exit(0))
+			//})
+			//Eventually(helpers.CF("bind-service", appName1, managedService1)).Should(Exit(0))
+			//Eventually(helpers.CF("bind-service", appName1, managedService2)).Should(Exit(0))
+			//Eventually(helpers.CF("bind-service", appName1, userProvidedService1)).Should(Exit(0))
+			//Eventually(helpers.CF("bind-service", appName1, userProvidedService2)).Should(Exit(0))
+			//Eventually(helpers.CF("bind-service", appName2, managedService2)).Should(Exit(0))
+			//Eventually(helpers.CF("bind-service", appName2, userProvidedService2)).Should(Exit(0))
```

Comment:
are these comments intentional?

Reply:
Yes because we will want to uncomment them once we can display bound apps. However I'm happy to remove them if you think it's confusing.